app/src/main/java/com/github/onlynotesswent/MainActivity.kt

```diff
@@ -89,132 +93,161 @@ fun OnlyNotesApp(
   val deckViewModel: DeckViewModel = viewModel(factory = DeckViewModel.Factory)
   val flashcardViewModel: FlashcardViewModel = viewModel(factory = FlashcardViewModel.Factory)
 
-  NavHost(navController = navController, startDestination = Route.AUTH) {
-    navigation(
-        startDestination = Screen.AUTH,
-        route = Route.AUTH,
-    ) {
-      composable(Screen.AUTH) { SignInScreen(navigationActions, userViewModel, authenticator) }
-      composable(Screen.CREATE_USER) { CreateUserScreen(navigationActions, userViewModel) }
-    }
+  val isInitialized = remember { mutableStateOf(false) }
```

Comment:
Given this is a remember, guarantee that it is reset after we log out (and thus go back to auth screen)

Reply:
`isInitialized` is used only in the beginning when the app is first launched, and its purpose is to delay the navigation until the `LaunchedEffect` has finished. Afterwards, it is not needed anymore